src/overlays/actors/ovl_En_Hg/z_en_hg.c

```diff
@@ -1,3 +1,9 @@
+/*
+ * File: z_en_hg.c
+ * Overlay: En_Hg
+ * Description: Pamela's Father (Normal)
```

Comment:
Normal seems a little ambiguous. I'd prefer healed to be a little more explicit.

Reply:
That is fair, I updated it to be Human if that makes it a little clearer